### PR TITLE
Critical upgrade known issue for RNs

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1829642

### CODE REVIEW

**Comment by codyhoag** [comment]
@mffiedler can you confirm this OCP 4.4 upgrade issue for the release notes? Thanks!

**Comment by mffiedler** [comment]
Looks good, but there is still a late-breaking effort to get a fix in.   I would hold off on merging this until we see what develops there.

**Comment by codyhoag** [comment]
@mffiedler sure thing; I'll hold this just in case we need it. Thanks!

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -2110,7 +2110,17 @@ issue by deleting the Elasticsearch deployments in sequence.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1824006[*BZ#1824006*])
 
 * {product-title} 4.4 is not shipping with a v.4.4 Metering Operator. Customers can install or continue to run the v.4.3 Metering Operator on {product-title} 4.4 clusters.
-(link:https://bugzilla.redhat.com/show_bug.cgi?id=1829035[*BZ1829035*])
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1829035[*BZ#1829035*])
+
+* After upgrading an {product-title} 4.1 or 4.2 cluster to version 4.4, the
+MachineSets cannot scale up because they still reference the previous versioned
+cluster's boot image (link:https://bugzilla.redhat.com/show_bug.cgi?id=1829642)[*BZ#1829642*]).
```
Comment (bergerhoffer) [inline release_notes/ocp-4-4-release-notes.adoc:2117]: The period on this is after the BZ link, where other known issues have them before the BZ link.

And right or wrong, other known issues that have workaround steps put the BZ link after the workaround steps.

**Review by bergerhoffer — COMMENTED** [review]
Just a small thing or two.

File: release_notes/ocp-4-4-release-notes.adoc (hunk 1 above)
Comment (codyhoag) [reply release_notes/ocp-4-4-release-notes.adoc:2117]: Good point; I'll add it to the end for consistency. Thanks!

**Comment by codyhoag** [comment]
@mffiedler suggested earlier that this likely wouldn't be needed. I won't merge this in unless I hear otherwise.

**Comment by mffiedler** [comment]
Confirming, we can drop this RN

**Pull request closed without merging**